pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -299,13 +298,28 @@ func (lbaas *LbaasV2) GetLoadBalancer(clusterName string, service *api.Service)
 	return status, true, err
 }
 
+func nodeAddressForLB(node *api.Node) (string, error) {
+	addrs := node.Status.Addresses
+	if len(addrs) == 0 {
+		return "", ErrNoAddressFound
+	}
+
+	for _, addr := range addrs {
+		if addr.Type == api.NodeInternalIP {
```

Comment:
can you verify how the kubelet populates this field? is there a possibility that it plugs in the ip of an interface that the openstack lb can't reach?